### PR TITLE
corrected/clarified references

### PR DESCRIPTION
**Pull request by cedrict**
Pull Request Checklist. Please read and check each box with an X. Delete any part not applicable. Ask on the [forum](https://community.geodynamics.org/c/aspect) if you need help with any step.

The references to 4 papers in grain_size.cc were not in the same style and they did not contain a doi link. This PR fixes that.

### Before your first pull request:

* [x ] I have read the guidelines in our [CONTRIBUTING.md](../blob/master/CONTRIBUTING.md) document.

### For all pull requests:

* [x ] I have followed the [instructions for indenting my code](../blob/master/CONTRIBUTING.md#making-aspect-better).

### CODE REVIEW

**Comment by tjhei** [comment]
please fix indentation.

**Comment by cedrict** [comment]
just ran make indent. should have fixed the problem.

File: source/material_model/grain_size.cc [hunk 1]
```diff
@@ -1042,10 +1042,10 @@ namespace aspect
                              "Units: \\si{\\meter}.");
           prm.declare_entry ("Use paleowattmeter", "true",
                              Patterns::Bool (),
-                             "A flag indicating whether the computation should be use the "
-                             "paleowattmeter approach of Austin and Evans (2007) for grain size reduction "
-                             "in the dislocation creep regime (if true) or the paleopiezometer approach "
-                             "from Hall and Parmetier (2003) (if false).");
+                             "A flag indicating whether the computation should be use the paleowattmeter approach of "
```
Comment (bangerth) [inline source/material_model/grain_size.cc:1045]:
```suggestion
                             "A flag indicating whether the computation should use the paleowattmeter approach of "
```

File: source/material_model/grain_size.cc [hunk 2]
```diff
@@ -1042,10 +1042,10 @@ namespace aspect
...
+                             "Austin and Evans (2007) \\url{http://dx.doi.org/10.1130/G23244A.1} for grain size reduction "
```
Comment (bangerth) [inline source/material_model/grain_size.cc:1046]: I think a better approach would be to put the full reference into `doc/manual/manual.bib` and `\\cite{...}` it here. Same for Hall and Parmentier a couple of lines below.

File: source/material_model/grain_size.cc [hunk 3]
```diff
@@ -1482,18 +1482,13 @@ namespace aspect
                                    "In the diffusion creep regime, the viscosity depends "
                                    "on this grain size field. "
                                    "We use the grain size evolution laws described in Behn "
-                                   "et al., 2009. Implications of grain size evolution on the "
-                                   "seismic structure of the oceanic upper mantle, "
-                                   "Earth Planet. Sci. Letters, 282, 178–189. "
+                                   "et al. (2009) \\url{http://dx.doi.org/10.1016/j.epsl.2009.03.014}. "
```
Comment (bangerth) [inline source/material_model/grain_size.cc:1485]: and here

File: source/material_model/grain_size.cc [hunk 4]
```diff
@@ -1482,18 +1482,13 @@ namespace aspect
...
                                    "to the 'simple' material model, or read from data files "
                                    "that were generated by the Perplex or Hefesto software. "
                                    "This material model "
-                                   "is described in more detail in Dannberg, J., Z. Eilon, "
-                                   "U. Faul, R. Gassmoeller, P. Moulik, and R. Myhill (2017), "
-                                   "The importance of grain size to mantle dynamics and "
-                                   "seismological observations, Geochem. Geophys. Geosyst., "
-                                   "18, 3034–3061, doi:10.1002/2017GC006944.")
+                                   "is described in more detail in Dannberg et al. (2017) "
+                                   "\\url{http://dx.doi.org/10.1002/2017GC006944}.")
```
Comment (bangerth) [inline source/material_model/grain_size.cc:1491]: and here.

Comment (bangerth) [reply source/material_model/grain_size.cc:1491]: this one might actually already be in the .bib file

**Comment by bangerth** [comment]
@cedrict -- ping?

**Comment by bangerth** [comment]
Did you close this on purpose?

**Comment by cedrict** [comment]
oh, I was doing some cleaning this afternoon ... oups

**Pull request closed without merging**